specification/servicelinker/resource-manager/Microsoft.ServiceLinker/preview/2021-11-01-preview/servicelinker.json

```diff
@@ -615,6 +623,14 @@
           "readOnly": true,
           "type": "string",
           "description": "The provisioning state. "
+        },
+        "vNetSolution": {
```

Comment:
According to Python log, there is duplicated object definition, please check it.
![image](https://user-images.githubusercontent.com/70930885/153116345-a41a5c2d-97f6-4e45-b9db-63c428f016b9.png)